docs/jobs.adoc

```diff
@@ -0,0 +1,89 @@
+// vim: tw=79
+= Jobs
+:toc:
+
+== List jobs
+
+List all jobs created.
+
+Sample Request
+
+----------
+curl -XGET
+http://127.0.0.1/api/1.0/jobs
+----------
+
+Sample Response
+
+----------
+Status: 200 OK
+[{
+	"integration_id": "717315a1-2a49-4cb7-826b-d2aa3bc4721a",
```

Comment:
The job_id will be in response to ImportCluster for ex. which can be used the query the job with the Jobs Details API